### PR TITLE
cmd/atlas: extend and fix changes from sql statements

### PR DESCRIPTION
**Pull request by a8m**

### CODE REVIEW

File: cmd/atlas/internal/sqlparse/myparse/myparse_test.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+		schema.Changes{
+			&schema.ModifyTable{
+				Changes: schema.Changes{
+					&schema.RenameColumn{From: schema.NewColumn("c1"), To: schema.NewColumn("c2")},
+				},
+			},
+		},
+		changes,
+	)
+
```
Comment (masseelch) [inline cmd/atlas/internal/sqlparse/myparse/myparse_test.go:52]: :rocket:

File: cmd/atlas/internal/sqlparse/sqliteparse/sqliteparse.go [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+	lex.AddErrorListener(l)
+	p := NewParser(
+		antlr.NewCommonTokenStream(lex, 0),
+	)
+	p.RemoveErrorListeners()
+	p.AddErrorListener(l)
+	p.BuildParseTrees = true
+	stmt = &Stmt{
+		p:    p,
+		stmt: p.Sql_stmt(),
```
Comment (masseelch) [inline cmd/atlas/internal/sqlparse/sqliteparse/sqliteparse.go:70]: :no_mouth:

File: cmd/atlas/internal/sqlparse/sqlparse.go [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	// A nop analyzer.
+	return FixerFunc(func(_ string, changes schema.Changes) (schema.Changes, error) {
+		return changes, nil
+	})
+}
+
+func init() {
+	Register(mysql.DriverName, FixerFunc(myparse.FixChange))
+	Register(postgres.DriverName, FixerFunc(pgparse.FixChange))
+	Register(sqlite.DriverName, FixerFunc(sqliteparse.FixChange))
```
Comment (masseelch) [inline cmd/atlas/internal/sqlparse/sqlparse.go:56]: Shoudln't this be done by the different implementations and not alongside the interface definition? Or this meant to be for "outsiders" to add their own fixers?

File: cmd/atlas/internal/sqlparse/sqliteparse/sqliteparse.go (hunk 2 above)
Comment (a8m) [reply cmd/atlas/internal/sqlparse/sqliteparse/sqliteparse.go:70]: Yeah, I know.

**Review by masseelch — APPROVED** [review]
Awesome (and ugly :laughing:)

**Pull request merged**